opennms-webapp-rest/src/main/java/org/opennms/web/rest/v2/EventConfPersistenceService.java

```diff
@@ -188,4 +218,20 @@ public Map<String, Object> filterEventConfSource(String filter, String sortBy, S
         return eventConfSourceDao.filterEventConfSource(filter, sortBy, order, totalRecords, offset, limit);
     }
 
+    private Event toXmlEvent(EventConfEventRequest dto) {
```

Comment:
we should just unmarshal from whole `xmlContent` and create `org.opennms.netmgt.xml.eventconf.Event`

and then set EventConfEvent from this object.